jbehave-support-core/docs/Reporting.md

```diff
@@ -32,10 +32,15 @@ There are several extensions already prepared and ready to use:
  - `WsXmlReporterExtension` (copies SOAP requests/responses from [WebServiceSteps](Web-service.md))
  - `TestContextXmlReporterExtension` (copies contents of [TestContext](Test-context.md))
  - `ServerLogXmlReporterExtension` (copies server log(s) for each system with configured [SshTemplate](Ssh.md))
+ - `SqlXmlReporterExtension` (copies SQL statements/results from [SqlSteps](Sql-steps.md))
  - `FailScreenshotsReporterExtension` (prints out error screenshots from [Web testing](Web-testing.md) - if any were generated)
  - `StepScreenshotsReporterExtension` (prints out step (not error) screenshots from [Web testing](Web-testing.md) - if any were generated)
- - `SqlXmlReporterExtension` (copies SQL statements/results from [SqlSteps](Sql-steps.md))
-
+   - Frequency of screenshot taking can be controlled by property: 'web.screenshot.type'
```

Comment:
Just a question about naming, this is more or less a reporting feature so how about - web.screenshot.reporting? (Or web.screenshot.reporting.type but that seems too long, or web.screenshot.reporting.mode)
I am not sure myself what seems better 🙂

Reply:
Good idea :)